src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -25,27 +24,34 @@ public class CrossTenantQueryService {
   private final PermissionsService permissionsService;
 
   private static final String COMPOSITE_INSTANCES_ID = "6b08439b-4f8e-4468-8046-ea620f5cfb74";
+  private static final String SIMPLE_INSTANCES_ID = "8fc4a9d2-7ccf-4233-afb8-796911839862";
 
   public List<String> getTenantsToQuery(EntityType entityType, boolean forceCrossTenantQuery) {
     if (!forceCrossTenantQuery && !Boolean.TRUE.equals(entityType.getCrossTenantQueriesEnabled())) {
       return List.of(executionContext.getTenantId());
     }
-    List<Map<String, String>> userTenantMaps = getUserTenants();
-    String centralTenantId = getCentralTenantId(userTenantMaps); // null if non-ECS environment; non-null otherwise
+    // Get the user's primary affiliation first, since this comes from mod-users and should work in non-ECS environments
+    // We can use this for determining if it's an ECS environment, and if so, retrieving the consortium ID and central tenant ID
+    Map<String, String> primaryAffiliation = getPrimaryAffiliation();
```

Comment:
This should probably all get reworded/renamed, since it's not actually getting the user's primary affiliation anymore